### ISSUE TITLE
Notification sending the same message to requesters/observers when emails are sent within the same minute – issue with ##followup.description##

### ISSUE THREAD

**Issue by prutkowski-cd** [glpi-project/glpi#18371]
### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Version

10.0.16

### Bug description

We are experiencing an issue with follow-up messages (##followup.description##) where multiple follow-up messages sent within the same minute (queuednotification run frequency - 1 minute) for the same ticket result in the first follow-up message being sent multiple times in email notifications to the requester and observers.

Here is the expected behavior:

- Each distinct follow-up message should trigger a separate email notification, containing the specific content of each follow-up.

However, the actual behavior is:

- When multiple follow-up messages are sent within the same minute for the same ticket, two identical email notifications are sent, both containing only the first follow-up message, while no email notification is sent for the second follow-up message or any subsequent messages.

This issue seems to occur when follow-up messages are added within the same minute, causing the notification system to multiply the first message in multiple notifications.

### Relevant log output

```shell

```

### Steps To reproduce

0. [Configured ticket system in GLPI with notifications → email template shown below]
1. Send an email to the configured mail receiver in GLPI.
2. After receiving the notification of a new ticket creation, reply to add a follow-up message.
3. Add a second, different follow-up message within the same minute.
4. Check in GLPI if it received both messages at the same time. If not, repeat steps 2 and 3.
5. Check if the mail client (receiver/observer) receives two identical email notifications, both containing the first follow-up message twice, but no email with the second follow-up message.

### Anything else?

Plugins:
-MailAnalyzer - enabled and used
-Ticketfilter - not used and disabled (was once enabled to check options)

Email Template in GLPI:

  ##FOREACH LAST followups##
  Ticket Title: ##ticket.title##
  
  Requesters: ##ticket.authors##
  Observers: ##ticket.observerusers##
  ##IFticket.assigntousers##
  Technicians: ##ticket.assigntousers##
  ##ENDIFticket.assigntousers## ##ELSEticket.assigntousers##
  Technicians: None assigned
  ##ENDELSEticket.assigntousers##
  
  ##IFfollowup.author.email##
  [##followup.date##] ##followup.author.email## wrote:
  
  ##ENDIFfollowup.author.email##
  ##followup.description##
  
  ##ENDFOREACHfollowups##

**Comment by prutkowski-cd** [comment]
I have verified whether the issue is related to plugins, and it is not. I restored the virtual machine to a state prior to using any plugins, and the same problem persists. It seems the issue is connected to the way notifications are processed, as all messages are correctly displayed in GLPI, but sometimes notifications sent within the same minute are sent incorrectly.

**Comment by cedric-anne** [comment]
It is probably fixed by #18922.

**Comment by prutkowski-cd** [comment]
Yes, I confirm that this fix resolves the issue. Thank you for your help.

Best regards,
